Serializer/Handler/ResourceCollectionHandler.php

```diff
@@ -1,59 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony CMF package.
- *
- * (c) 2011-2014 Symfony CMF
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Cmf\Bundle\ResourceRestBundle\Serializer\Handler;
-
-use Puli\Repository\Api\ResourceCollection;
-use Puli\Repository\Api\Resource\Resource;
-use JMS\Serializer\Handler\SubscribingHandlerInterface;
-use JMS\Serializer\GraphNavigator;
-use JMS\Serializer\JsonSerializationVisitor;
-use JMS\Serializer\Context;
-
-/**
- * Serialize ResourceCollection instances into flat arrays
- *
- * @author Daniel Leech <daniel@dantleech.com>
- */
-class ResourceCollectionHandler implements SubscribingHandlerInterface
```

Comment:
We no longer need  a handler for resource collections as we handle this in the ResourceHandler.